.gitignore

```diff
@@ -1,9 +1,12 @@
+openfeature*
```

Comment:
Why was this added to the gitignore?

Reply:
This was to avoid certificates being committed, I'll remove it.